Binary files nextcloud-client-2.4.0.orig/.git/index and nextcloud-client-2.4.0/.git/index differ
diff -ruN nextcloud-client-2.4.0.orig/src/CMakeLists.txt nextcloud-client-2.4.0/src/CMakeLists.txt
--- nextcloud-client-2.4.0.orig/src/CMakeLists.txt	2018-02-01 19:16:38.461162563 +0100
+++ nextcloud-client-2.4.0/src/CMakeLists.txt	2018-02-01 19:20:20.253824030 +0100
@@ -4,7 +4,7 @@
 
 set(synclib_NAME ${APPLICATION_EXECUTABLE}sync)
 
-find_package(Qt5 5.6 COMPONENTS Core Network Xml Concurrent WebEngineWidgets WebEngine REQUIRED)
+find_package(Qt5 5.5 COMPONENTS Core Network Xml Concurrent WebEngineWidgets WebEngine REQUIRED)
 if (Qt5Core_VERSION VERSION_LESS 5.9.0)
 message(STATUS "For HTTP/2 support, compile with Qt 5.9 or higher.")
 endif()
--- nextcloud-client-2.4.0.orig/admin/osx/CMakeLists.txt	2018-05-22 07:01:48.248646951 +0200
+++ nextcloud-client-2.4.0/admin/osx/CMakeLists.txt	2018-05-22 07:01:51.012689711 +0200
@@ -11,7 +11,7 @@
   set(MAC_INSTALLER_DO_CUSTOM_BACKGROUND "0")
 endif()
 
-find_package(Qt5 5.6 COMPONENTS Core REQUIRED)
+find_package(Qt5 5.5 COMPONENTS Core REQUIRED)
 configure_file(create_mac.sh.cmake ${CMAKE_CURRENT_BINARY_DIR}/create_mac.sh)
 configure_file(macosx.pkgproj.cmake ${CMAKE_CURRENT_BINARY_DIR}/macosx.pkgproj)
 configure_file(pre_install.sh.cmake ${CMAKE_CURRENT_BINARY_DIR}/pre_install.sh)
